schemas/compat-data-schema.md

```diff
@@ -252,7 +252,7 @@ Examples:
 ### Ranged versions
 
 For certain browsers, ranged versions are allowed as it is sometimes impossible to find out in which early version of a browser a feature shipped. The statement below means "supported in at least version 37 and probably in earlier versions as well".
-Currently, the only allowed ranged version is `"≤37"` for `webview_android`. There will be more ranged versions for other browsers in the future but ranged versions aren't generally allowed for every version string. Ranged versions should be used sparingly and only when it is impossible to find out the version number a feature initially shipped in.
+Currently, the only allowed ranged versions are `"≤37"` for `webview_android` and `"≤18"` + `"≤79"` for `edge`. There will be more ranged versions for other browsers in the future but ranged versions aren't generally allowed for every version string. Ranged versions should be used sparingly and only when it is impossible to find out the version number a feature initially shipped in.
```

Comment:
Since we're expanding the use of version ranges now, I think it makes sense to back away from the "currently" and "future" stuff. How about something like this:

> For certain browsers, ranged versions are allowed as it is sometimes impossible to find out in which early version of a browser a feature shipped. Ranged versions should be used sparingly and only when it is impossible to find out the version number a feature initially shipped in. The following ranged version values are allowed:
> * `edge`: `"≤18"`, `"≤79"`
> * `webview_android`: `"≤37"`
>
> 
> For example, the statement below means, "supported in at least version 37 and probably in earlier versions as well".

Reply:
I like that wording, yeah!

Reply:
I was about to propose changes, too. Maybe we could even explain "≤18", "≤79" (and "≤37") some more with what we have discussed in this issue? 
"≤18" (supported in some version of EdgeHTML Edge)
"≤79" (supported in some version Chromium and Edge and possibly in EdgeHTML Edge)
"≤37" (supported in former Android versions prior to Chrome-based WebView)